framework/Samplers/Sampler.py

```diff
@@ -792,6 +792,18 @@ def localFinalizeActualSampling(self,jobObject,model,myInput):
     """
     pass
 
+  def _reassignSampledVarsPbToFullyCorrVars(self):
```

Comment:
this method should be included in every new sampler right?
It would be worth mentioning it in the description

Reply:
I can add it